File: ICP/Assignment_7/Q1.java
/*Name-Anish More
  Registration number:2241013365
  Name of the program:Write a java program to create an array of size N and store the random values between 1 to N in it and find the sum and average.*/
import java.util.Scanner;
public class Q1 
{
	public static void main(String[] args) 
	{
		Scanner sc=new Scanner(System.in);
		System.out.print("Enter number of terms ");
		int n=sc.nextInt();
		int arr[]=new int[n];
		int sum=0;
		for(int i=0;i<n;i++)
		{
			arr[i]=(int)(Math.random()*n+1);
			sum+=arr[i];
		}
		System.out.println("Sum= "+sum);
		System.out.println("Average = "+(sum/n));
		
	}
}
	
/*Output
Enter number of terms 10
Sum= 66
Average = 6
 */

File: Python/Minor_Assignment_4/Q1.py
import random as r
n=int(input("Enter N:"))
L=[r.random()  for i in range(n)]
s=sum(L)
print("Random list: ",L)
print("Sum: %.2f"%s)
print("Average: %.2f"%(s/n))

'''
Output--
Enter N:5
Random list:  [0.2532507414408488, 0.751389081119467, 0.5014945614943557, 0.7102580845384576, 0.18894512208938008]
Sum: 2.41
Average: 0.48
'''